processing/src/main/java/io/druid/query/dimension/DimensionSpec.java

```diff
@@ -35,8 +35,7 @@
     @JsonSubTypes.Type(name = "default", value = DefaultDimensionSpec.class),
     @JsonSubTypes.Type(name = "extraction", value = ExtractionDimensionSpec.class),
     @JsonSubTypes.Type(name = "regexFiltered", value = RegexFilteredDimensionSpec.class),
-    @JsonSubTypes.Type(name = "listFiltered", value = ListFilteredDimensionSpec.class),
-    @JsonSubTypes.Type(name = "lookup", value = LookupDimensionSpec.class)
```

Comment:
why is this removed?

Reply:
I had to move this to https://github.com/a2l007/druid/blob/8f2a360119be5d619a2bc0d57976f483faaad610/server/src/main/java/io/druid/query/lookup/LookupModule.java#L89

as part of the refactoring.